src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -174,13 +177,15 @@ export class NativeStakingMapper {
    * native staking `batchWithdrawCLFee` transaction.
    *
    * @param args.chainId - the chain ID of the native staking deployment
+   * @param args.safeAddress - the Safe staking
```

Comment:
nit: We could include the argument type as well.
```suggestion
   * @param {string} args.safeAddress - the Safe staking
```

Reply:
Added in a2ddb8d0a00494f553f85b88871705788d468006.